src/test_files/discordTest.js

```diff
@@ -0,0 +1,12 @@
+const DiscordBot = require('../../discordBot.js')
+
+
+var testBot = new DiscordBot(process.env.DISCORD_BOT_TOKEN);
```

Comment:
var is deprecated, it's better to use const testBot = new DiscordBot(process.env.DISCORD_BOT_TOKEN);

Reply:
fixed